oci-info/action/oci.py

```diff
@@ -74,9 +74,10 @@ def __init__(
         super().__init__(
             base_url=f"https://{reg_domain}",
             timeout=timeout,
-            auth=(user, password) if user else None,
             headers={"accept": ",".join([h.value for h in OciMediaTypes])},
         )
+
+        self.user_auth = (user, password) if user else None
```

Comment:
```suggestion
        self.user_auth = (user, password) if user and password else None
```
Shouldn't you also check for password?

Reply:
sure